app/src/main/AndroidManifest.xml

```diff
@@ -38,6 +38,9 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".ui.stars.keyword.KeywordsActivity"
+            android:label="@string/title_keywords_activity"/>
```

Comment:
you are literally adding and space to the other strings but the one you add it doesn't have any space at the end?